src/storage/file_manager.c

```diff
@@ -4324,7 +4324,12 @@ file_temp_retire_internal (THREAD_ENTRY * thread_p, const VFID * vfid, bool was_
   else
     {
       entry = file_tempcache_pop_tran_file (thread_p, vfid);
-      assert (entry != NULL);
```

Comment:
Is there a state in which a temp file is not related to a transaction and not preserved? Why don't you preserve it?

Reply:
I fixed it to revert it.